src/server/pps/cmds/cmds.go

```diff
@@ -438,6 +439,24 @@ each datum.`,
 		follow      bool
 		tail        int64
 	)
+
+	// prettyLogsPrinter helps to print the logs recieved in different colours
+	prettyLogsPrinter := func(message string) {
+		informationArray := strings.Split(message, " ")
+		debugLevel := strings.ToLower(informationArray[1])
```

Comment:
This won't work with all logs, since they aren't always structured properly. As an example, try the opencv demo (`cd examples/opencv && make opencv`), then run `pachctl logs`; it'll panic:

```
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
github.com/pachyderm/pachyderm/src/server/pps/cmds.Cmds.func11(0xc000a9d1c0, 0x38)
	src/server/pps/cmds/cmds.go:446 +0x2da
github.com/pachyderm/pachyderm/src/server/pps/cmds.Cmds.func12(0x4d23540, 0x0, 0x0, 0x0, 0x0)
	src/server/pps/cmds/cmds.go:509 +0x494
github.com/pachyderm/pachyderm/src/server/pkg/cmdutil.RunFixedArgs.func1(0xc000716a00, 0x4d23540, 0x0, 0x0)
	src/server/pkg/cmdutil/cobra.go:30 +0x135
github.com/spf13/cobra.(*Command).execute(0xc000716a00, 0x4d23540, 0x0, 0x0, 0xc000716a00, 0x4d23540)
	/Users/yusuf/go/pkg/mod/github.com/spf13/cobra@v0.0.6-0.20191202130430-b04b5bfc50cb/command.go:833 +0x2aa
github.com/spf13/cobra.(*Command).ExecuteC(0xc0005f4f00, 0x0, 0x0, 0xc000823e30)
	/Users/yusuf/go/pkg/mod/github.com/spf13/cobra@v0.0.6-0.20191202130430-b04b5bfc50cb/command.go:917 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/yusuf/go/pkg/mod/github.com/spf13/cobra@v0.0.6-0.20191202130430-b04b5bfc50cb/command.go:867
main.main.func1(0x0, 0x0)
	src/server/cmd/pachctl/main.go:20 +0x55
main.main()
	src/server/cmd/pachctl/main.go:21 +0xbc
```

Reply:
I didn't get this error but I understand your concern, let me address that.